### ISSUE TITLE
Ability to commit offset of consumer

### ISSUE THREAD

**Issue by lironess** [confluentinc/kafka-rest-node#14]
Hi.
There is an option for the Kafka Rest Proxy to commit offsets as you can see [here](http://docs.confluent.io/2.0.0/kafka-rest/docs/api.html#post--consumers-%28string-group_name%29-instances-%28string-instance%29-offsets).
I want this options to be available for a consumer i created.
something like :

``` javascript
var kafkaRest = require('kafka-rest');

var kafka = new kafkaRest({ url: restProxyServerUrl });

kafka.consumer(myConsumerGroup).join(
  { 'format': `avro`, 'auto.offset.reset': `smallest`, `socket.timeout.ms`: `20000` }
  , function(error, consumer) {
    var listener = consumer.subscribe(topic);

    listener.on(`data`, function(messages) {
      consumer.commitOffset();
    });
  });
```

notice the `consumer.commitOffset();` - it is like making a HTTP request to [this](http://docs.confluent.io/2.0.0/kafka-rest/docs/api.html#post--consumers-%28string-group_name%29-instances-%28string-instance%29-offsets) with the consumerGroup and raw name from the instance.

**Comment by pcothenet** [comment]
+1

**Comment by yuanzhaoYZ** [comment]
+1

**Comment by killfill** [comment]
+1

**Comment by toaderflorin** [comment]
Is there any update on this? From what I see, there isn't support for this in the library.

**Comment by ewencp** [comment]
Currently this isn't implemented, but if someone wants to file a patch the implementation should be a very simple implementation that just makes a request to the [offset commit API](http://docs.confluent.io/3.1.1/kafka-rest/docs/api.html#post--consumers-(string-group_name)-instances-(string-instance)-offsets). It'd probably only be a few lines of code.

**Comment by toaderflorin** [comment]
OK, created a pull request for that.

**Comment by ewencp** [comment]
Fixed by #25
